src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.Bindings.Core.targets

```
@@ -44,23 +44,33 @@ It is shared between "legacy" binding projects and .NET 5 projects.
     />
   </Target>
 
+  <Target Name="_CollectLibrariesToBind">
+    <ItemGroup>
+      <_LibrariesToBind  Include="@(AndroidLibrary)" Condition=" '%(AndroidLibrary.Extension)' == '.jar' And '%(AndroidLibrary.Bind)' == 'true' " />
+      <_LibrariesToBind  Include="@(AndroidLibrary)" Condition=" '%(AndroidLibrary.Extension)' == '.aar' And '%(AndroidLibrary.Bind)' == 'true' " />
+      <_LibrariesToBind  Include="@(EmbeddedJar)" />
+      <_LibrariesToBind  Include="@(InputJar)" />
+      <_LibrariesToBind  Include="@(LibraryProjectZip)" />
+      <_LibrariesToBind  Include="@(_JavaBindingSource)" Condition=" '%(_JavaBindingSource.Bind)' == 'true' "/>
+    </ItemGroup>
+  </Target>
```

I was thinking about this target that "categorizes" the `@(AndroidLibrary)` item group:

https://github.com/xamarin/xamarin-android/blob/c20d51fcf8e910b8fb46c5351c26e55ed1fab90c/src/Xamarin.Android.Build.Tasks/MSBuild/Xamarin/Android/Xamarin.Android.AvailableItems.targets#L64-L87

What if we did:

```suggestion
  <Target Name="_CollectLibrariesToBind" DependsOnTargets="_CategorizeAndroidLibraries">
    <ItemGroup>
      <_LibrariesToBind  Include="@(EmbeddedJar)" />
      <_LibrariesToBind  Include="@(InputJar)" />
      <_LibrariesToBind  Include="@(LibraryProjectZip)" />
      <_LibrariesToBind  Include="@(_JavaBindingSource)" Condition=" '%(_JavaBindingSource.Bind)' == 'true' "/>
    </ItemGroup>
  </Target>
```

We'd then not specify `@(AndroidLibrary)` here? We'd only use the older item groups names like the original `Condition` was doing before.